client/src/js/views/DashboardDogsListDogViewTemplate.html

```diff
@@ -1,4 +1,4 @@
-<div class="dog dog--dashboard">
+<div class="dog dog--dashboard <% if (checked_in == false) { %> dog--not-checked-in <% } %>">
```

Comment:
Let's reverse this for brevity and clarity. Instead of `.dog--not-checked-in` just do `.dog--checked-in`